book/website/collaboration/inclusive-events/inclusive-events-schedules.md

```diff
@@ -0,0 +1,286 @@
+
+Although it is becoming more common for people with lived experience to become self-advocates and receive training in public speaking, it is important for researchers and event organizers to consider, and make use of, moderators and session facilitators in order to uphold inclusive and equitable engagement practices. 
+
+(cl-inclusive-events-icebreakers)=
+### Using Icebreakers and Teambuilding
+Additionally, you may consider using a collaborative activity to open your session such as an icebreaker or another team-building task. 
+This can help build rapport among event attendees, and orientate everyone towards a shared goal before the formal parts of your session begin.
+
+By establishing an active partnership between people with lived experience, researchers, and other experts by profession in your event's sessions, you will be well-positioned to support an equitable space for all attendees to meaningfully engage with your event.
+
+(cl-inclusive-events-inclusive-program-planning)=
+## Creating an Inclusive Program
+If you are able, consider including a person with lived experience in the planning of an event's program and agenda. 
+
+Involving people with lived experience in the planning and delivery of events ensures the voice of people with lived experience is built into the dialogue throughout your event. 
+
+Active involvement can also help organizers develop a greater understanding of what is important from the perspective of people with lived experience and ensure that the proposed structure and content of the event is shaped in a way that will meet the needs of the potential attendees.
+
+Their involvement can offer insight and perspective into the specific challenges and needs of individuals with disabilities and any specific lived experience your event is focused on. If you are including a person, or group, with lived experience in the planning, ensure that you are appropriately compensating and supporting them for their expertise.
+
+If you are unable to include a representative on your planning team, try to prioritize the engagement of, and feedback from, the specific communities that will be in attendance. People with lived experience may offer different ideas about how to present the materials, and their insights will be invaluable in developing an inclusive event program that is genuinely accessible, respectful, and responsive to the diverse needs of all participants.
+
+When developing your event's program or agenda, ensure that you have established a pathway or mechanism to get feedback and input from people with lived experience. The two options discussed above are ideal, as people with lived experience will be able to provide insights and perspectives on the specific challenges and needs of your attendees. This perspective can be helpful when identifying what sessions or content should be included in the program, and will support you in creating a more inclusive and relevant program for your audience. 
```

Comment:
```suggestion
When developing your event's program or agenda, ensure that you have established a pathway or mechanism to get feedback and input from people with lived experience. 
To receive this feedback and input you can include people with lived experience in the planning committee of your event, and you can ask potential attendees for feedback on the planning and materials of your event. 
```
I would add this as the first sentence of this section, so you introduce a bit better what this section is about. My suggested sentence probably needs an update...

Reply:
Made some changes :)